docs/site/src/components/AppFrame.js

```diff
@@ -171,7 +171,7 @@ class AppFrame extends Component {
           docked={drawerDocked}
           routes={routes}
           onRequestClose={this.handleDrawerClose}
-          open={this.state.drawerOpen}
+          open={drawerDocked || this.state.drawerOpen}
```

Comment:
Previously this worked, since now fixed bug introduced "Drawer is always open when docked"